package-lock.json

```diff
@@ -9068,45 +9068,98 @@
         "yarn-install": "^1.0.0"
       },
       "dependencies": {
+        "@wdio/config": {
```

Comment:
not sure how the lockfiles got changed? Did you try to upgrade wdio deps?

Reply:
yes, a leftover of a frustrated attempt to upgrade wdio deps 😄  thanks for catching it